src/docker/ContextManager.ts

```diff
@@ -96,8 +103,12 @@ export class DockerContextManager implements ContextManager, Disposable {
 
             // Create a new client
             if (currentContext.Type === 'aci') {
+                await this.setVsCodeContext('vscode-docker:aciContext', true);

```

Comment:
Right now, `vscode-docker:aciContext` and `vscode-docker:newSdkContext` mean the same thing, but that probably will not be true in the future, so I set it up as two different contexts.

Reply:
Put this as a comment in the code please